.azure-pipelines/client.yml

```diff
@@ -0,0 +1,62 @@
+trigger:
+- master
+
+jobs:
+- job: 'Build_Target_Packages'
+
+  pool:
+    vmImage: 'ubuntu-16.04'  
+
+  steps:
+  - task: UsePythonVersion@0
+    displayName: 'Use Python 3.6'
+    inputs:
+      versionSpec: 3.6
+
+  - script: |
+     python -m venv .\
+     
+     pip install git+https://github.com/lmazuel/swagger-to-sdk
+     pip install -e "git+https://github.com/Azure/azure-sdk-for-python#subdirectory=azure-sdk-tools&egg=azure-sdk-tools"
```

Comment:
Part of the `python scripts/dev_setup.py`, don't do it explicitly

Reply:
I can rip this out no issue due to your above comment. Will Remove and test.